CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -0,0 +1,72 @@
+# Changelog 1.11
+
+## 1.11.0 - 2023-03-22
+
+- The EndpointSlice feature is graduated from Alpha to Beta and is therefore enabled by default.
+
+### Added
+
+- Add the following capabilities to Antrea-native policies:
+  * ClusterSet scoped policy rule is now compatible with `namespaces` field. ([#4571](https://github.com/antrea-io/antrea/pull/4571), [@Dyanngg])
+  * Layer 7 policy rule is now compatible with `enableLogging` field. ([#4625](https://github.com/antrea-io/antrea/pull/4625), [@qiyueyao])
+  * FQDN policy rule now supports clients using DNS over TCP. ([#4612](https://github.com/antrea-io/antrea/pull/4612), [@GraysonWu])
+- Add the following capabilities to the Multi-cluster feature:
+  * Support "noEncap", "hybrid", and "networkPolicyOnly" in-cluster traffic encapsulation modes with Multi-cluster Gateway. ([#4407](https://github.com/antrea-io/antrea/pull/4407), [@luolanzone])
```

Comment:
"in-cluster traffic encapsulation modes" -> "Antrea traffic mode"

Reply:
Would it mislead reader the Multi-cluster Gateway could use noEncap, hybrid mode for cross-cluster traffic?
"Support "noEncap", "hybrid", and "networkPolicyOnly" Antrea traffic mode with Multi-cluster Gateway."